reservationWeekday.js

```diff
@@ -0,0 +1,18 @@
+var http = require('http');
+
+var hallsModule = require('./halls');
+var luxon = require('luxon');
+
+var provjera = hallsModule.halls.filter(hall => hall.reservation.isReserved === true);
```

Comment:
Kao naziv varijable je uvijek bolje staviti da bude ono što ta varijabla i predstavlja. Tako kod postane čitljiviji